Sources/tinys3/AWSRequest.swift

```diff
@@ -185,7 +93,9 @@ extension AWSRequest {
     }
 
     var canonicalQueryString: String {
-        queryItems.asEscapedQueryString
+        queryItems
+            .sorted { $0.name < $1.name }
+            .asEscapedQueryString
     }
```

Comment:
This fix was necessary otherwise the new `listMultipartUploadsRequest` failed to be HMAC-signed properly, and caused the request to be rejected.

This is because [signing a request with AWS4 requires the query parameters to be sorted alphabetically](https://docs.aws.amazon.com/IAM/latest/UserGuide/create-signed-request.html) — and with `listMultipartUploadsRequest` being the first request in our code so far that had its `URLQueryItems` not happen to be provided in alphabetical order, that's why we never noticed before.